data/data-pipeline/data_pipeline/etl/sources/mapping_for_ej/etl.py

```diff
@@ -0,0 +1,114 @@
+from data_pipeline.utils import get_module_logger
+from data_pipeline.score import field_names
+from data_pipeline.config import settings
+
+logger = get_module_logger(__name__)
+
+
+class MappingForEJETL(ExtractTransformLoad):
+    def __init__(self):
+        self.CSV_PATH = self.DATA_PATH / "dataset" / "mapping_for_ej"
+
+        self.MAPPING_FOR_EJ_VA_URL = (
+            settings.AWS_JUSTICE40_DATASOURCES_URL + "/VA_mej.zip"
+        )
+        self.MAPPING_FOR_EJ_CO_URL = (
+            settings.AWS_JUSTICE40_DATASOURCES_URL + "/CO_mej.zip"
+        )
+        self.VA_SHP_FILE_PATH = self.TMP_PATH / "mej_virginia_7_1.shp"
+        self.CO_SHP_FILE_PATH = self.TMP_PATH / "mej_colorado_final.shp"
+
+        # Defining variables
+        self.COLUMNS_TO_KEEP = [
+            self.GEOID_TRACT_FIELD_NAME,
+            field_names.MAPPING_FOR_EJ_FINAL_PERCENTILE_FIELD,
+            field_names.MAPPING_FOR_EJ_FINAL_SCORE_FIELD,
+            field_names.MAPPING_FOR_EJ_PRIORITY_COMMUNITY_FIELD,
+        ]
+
+        # Choosing constants.
+        # In our current score implementation, about 17% of CO and 20% of VA tracts are
+        # identified as disadvantaged. Consequently, the rank-based threshold is 20%.
+        # Using the scores to calculate which are priority communities doesn't quite track
+        # with this distribution, and so I've opted to choose roughly 20% of both states.
+        self.MAPPING_FOR_EJ_PRIORITY_COMMUNITY_PERCENTILE_THRESHOLD = 80
+
+        self.df: pd.DataFrame
+
+    def extract(self) -> None:
+        logger.info("Downloading Mapping for EJ Data")
+        super().extract(
+            self.MAPPING_FOR_EJ_VA_URL,
+            self.TMP_PATH,
+        )
+        super().extract(
+            self.MAPPING_FOR_EJ_CO_URL,
+            self.TMP_PATH,
+        )
+
+    def transform(self) -> None:
+        logger.info("Transforming Mapping for EJ Data")
+
+        # Join (here, it's just concatenating) the two dataframes from
+        # CO and VA, and drop any row that has a null identifier
+        self.df = pd.concat(
+            [
+                gpd.read_file(self.VA_SHP_FILE_PATH),
+                gpd.read_file(self.CO_SHP_FILE_PATH),
+            ]
+        )
+
+        # Fill Census tract to get it to be 11 digits, incl. leading 0s
+        # Note that VA and CO should never have leading 0s, so this isn't
+        # strictly necessary, but if in the future, there are more states
+        # this seems like a reasonable thing to include.
+        self.df[self.GEOID_TRACT_FIELD_NAME] = (
+            self.df["fips_tract"].astype(str).str.zfill(11)
+        )
+
+        self.df = self.df.rename(
+            columns={
+                "fin_rank": field_names.MAPPING_FOR_EJ_FINAL_PERCENTILE_FIELD,
+                "fin_score": field_names.MAPPING_FOR_EJ_FINAL_SCORE_FIELD,
+            }
+        )
+
+        # Note that there are tracts in this dataset that do not have a final ranking
```

Comment:
I added this, but I'm wavering. The issue is that not every single tract has complete data in this dataset, and as a consequence, not every tract has a final score or percentile. This isn't a huge deal -- it's a handful of tracts -- but I don't love labeling the tracts as "not prioritized" simply because they lacked complete data. Consequently, I dropped them. The other two options -- allow null tracts not to be prioritized or have the prioritization column be `NaN`, `True`, or `False` -- seemed less foolproof to me, but I don't have particularly strong feelings here. @lucasmbrown-usds

Reply:
Eh, to be consistent with previous ETLs, this does not happen 👍